views/default/user.html

```diff
@@ -32,9 +32,32 @@
             {{else: }}
                 <div class="carousel slide col-md-12" data-ride="carousel" id="myCarousel"  style="margin-bottom: 50px;">
             {{ pass }}
-            <p style="text-align: center;">
-                <img src="/static/images/logo_small.png" width="400" style="margin: auto;" >
-            </p>
+                <!-- Indicators -->
```

Comment:
This seems to undo my earlier changes where I removed the carousel and updated outdated links.